src/coreclr/pal/src/exception/remote-unwind.cpp

```diff
@@ -1381,6 +1381,163 @@ StepWithCompactEncodingRBPFrame(const libunwindInfo* info, compact_unwind_encodi
     return true;
 }
 
+static bool
+StepWithCompactEncodingFrameless(const libunwindInfo* info, compact_unwind_encoding_t compactEncoding, unw_word_t functionStart)
+{
+    int mode = compactEncoding & UNWIND_X86_64_MODE_MASK;
```

Comment:
Can you please add the comment here this came from `https://github.com/llvm/llvm-project/blob/3e8c4366e223ebe6005316930f382efb251f8434/lldb/source/Symbol/CompactUnwindInfo.cpp#L801`, add the following disclaimer to the file:

```
//===-- CompactUnwindInfo.cpp ---------------------------------------------===//
//
// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
// See https://llvm.org/LICENSE.txt for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
//
//===----------------------------------------------------------------------===//
```

And add the license https://llvm.org/LICENSE.txt to the TPN file.